lib/web/userevent.go

```diff
@@ -124,6 +130,24 @@ func (h *Handler) createPreUserEventHandle(w http.ResponseWriter, r *http.Reques
 				Username: req.Username,
 			},
 		}
+	case recoveryCodesCopyClickEvent:
+		typedEvent.Event = &v1.UsageEventOneOf_UiRecoveryCodesContinueClick{
```

Comment:
same as below, should this be `v1.UsageEventOneOf_UiRecoveryCodesCopyClickEvent`?